.github/workflows/daily.yml

```diff
@@ -30,9 +30,9 @@ jobs:
         with:
           version: "0.36.13"
 
-      - name: Audit
-        run: cargo make --profile github-actions audit
-
+      #- name: Audit
+      #  run: cargo make --profile github-actions audit --ignore "RUSTSEC-2022-0093","RUSTSEC-2022-0090","RUSTSEC-2020-0016","RUSTSEC-2020-0056","RUSTSEC-2021-0145"
```

Comment:
Can you remove the flags so that this is just commenting out the original line? I assume those flags didn't work out.

Reply:
completed PR and left oriignal command